source/tsconfig-json.d.ts

```diff
@@ -713,7 +713,7 @@ declare namespace TsConfigJson {
 		/**
 		Specify path mapping to be computed relative to baseUrl option.
 		*/
-		paths?: Record<string, string[]>;
+		paths?: Partial<Record<string, string[]>>;
```

Comment:
Why is this changed?


VSCode doesn't allow `undefined` to be set:
<img width="480" alt="Screen Shot 2022-06-07 at 5 00 50 PM" src="https://user-images.githubusercontent.com/1075694/172481843-f35b95f0-f6a2-4479-87c5-aa0cdca02678.png">


tsc requires a non-empty array value:

<img width="1108" alt="Screen Shot 2022-06-07 at 5 01 49 PM" src="https://user-images.githubusercontent.com/1075694/172481991-ff9a5e87-b2f8-4587-9022-d198cfc8689e.png">

Reply:
Revert PR: https://github.com/sindresorhus/type-fest/pull/404